pinot-core/src/main/java/org/apache/pinot/core/data/partition/ByteArrayPartitionFunction.java

```diff
@@ -55,5 +52,9 @@ public int getNumPartitions() {
   public String toString() {
     return NAME;
   }
+
+  private int abs(int n) {
+    return (n == Integer.MIN_VALUE) ? 0 : Math.abs(n);
```

Comment:
Is this behavior the same as the original code?

Reply:
Verified it is with tests added